packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -196,6 +207,11 @@ abstract class ProjectBase extends Resource implements IProject {
     return this._connections;
   }
 
+  /**
+   * Enable batch builds.
+   */
+  public abstract enableBatchBuilds(): void
```

Comment:
This default implementation should _not_ be abstract; it should be concrete, and just return `undefined` (otherwise, you have to repeat this default implementation twice below).

Reply:
Cool. I actually did that initially but then switched it when I saw

```ts
  /** The IAM service Role of this Project. */
  public abstract readonly role?: iam.IRole;
```
which could also be non abstract and default to `undefined`